custom/icds_reports/static/icds_reports/mobile/css/app.less

```diff
@@ -315,7 +315,7 @@
 }
 .bar-chart {
   .bg-white;
-  height: 415px;
+  height: 100%;
```

Comment:
confirming you've tested this in other places the bar chart shows up?

Reply:
yes. Tested it